util/host_picker.py

```diff
@@ -0,0 +1,44 @@
+import random
+import requests
+
+# This assumes the format found in
+# https://raw.githubusercontent.com/mozilla/http-observatory-dashboard/master/httpobsdashboard/conf/sites.json
+# For instance:
+#{
+#  "sites": {
+#    "category1": {
+#      "site_name1": [
+#        "hostname1"
+#      ],
+#      "site_name2": [
+#        "hostname1",
+#        "hostname2",
+#      ],
+#    },
+#    "category2": {
+#      "site_name1": [
+#        "hostname1"
+#      ],
+#      "site_name2": [
+#        "hostname1",
+#        "hostname2",
+#      ],
+#    }
+#}
+#
```

Comment:
Perhaps we should simplify Randomizer class to accept a simple array rather than a proprietary JSON format?  I'm thinking it will be more general use that way.

Reply:
Agree, will change. But I am not even sure using randomizer is a good idea in the long run. I am thinking something like, read a bunch of hosts from a file, import to a list/array, then put all of them in a queue.
The file in question should be read periodically to see if there are newer entries, so they could be also put into the queue.

Reply:
ok